API/tests/Feature/ShoppingListControllerTest.php

```diff
@@ -15,7 +15,7 @@ class ShoppingListControllerTest extends TestCase
     // Runs our migrations to set up the in-memory database with all the proper tables
     use RefreshDatabase;
 
-    public function testCreateShoppingList()
+    public function create_validShoppingListTitle_savesShoppingListTitleToDatabase()
```

Comment:
No need for "Title" here, since it's clear that the GroceryItems for a ShoppingList are stored in a separate Model/table.